static/app/components/profiling/FlamegraphToolbar.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled';
+
+interface FlamegraphToolbarProps {
+  children: React.ReactNode;
+}
+
+export const FlamegraphToolbar = styled('div')<FlamegraphToolbarProps>`
```

Comment:
You don't need to specify the props here, unless the intent is to speed up the typescript compiler, if so I'm not sure if this makes a difference or not.

Reply:
Yeah, I did it just because I want to make the children type explicit, otherwise it's `children?: React.ReactNode`